site/app/templates/forum/HistoryForm.twig

```diff
@@ -4,7 +4,7 @@
 {% block body %}
     <div class="post_box" style="margin-left:0px;display: none;">
         <pre class="pre-forum"><span class="post_content" style="white-space: pre-wrap; "></span></pre>
-        <hr style="margin-bottom:3px;">
+        <hr style="margin-bottom:3px; margin-top:1.5em;">
```

Comment:
Is there an external stylesheet you can move this CSS (including the pre-existing CSS) to?